### ISSUE TITLE
load_vectors does not take keyword arguments?

### ISSUE THREAD

**Issue by jwohlwend** [pytorch/text#189]
Looking at the function definition for load_vectors, it doesn't seem to take optional arguments, so I'm wondering how line 206 in Vocab.py can work: 

```
self.load_vectors(vectors, unk_init=unk_init, expand_vocab=expand_vocab)
```

It would be nice if load vectors took all arguments to vector, so as to specify cache, or unk_init as above. Please let me know if I'm missing something..

**Comment by nelson-liu** [comment]
Yes, that looks like a bug --- I'm assuming @jekbradbury wrote that when there was a different API for `load_vectors`.

> It would be nice if load vectors took all arguments to Vectors, so as to specify cache, or unk_init as above. Please let me know if I'm missing something..

I'm a bit confused -- can you explain your rationale for doing this? Couldn't you just pass cache or `unk_init` when creating a `Vector` object, then pass the `Vector` to `load_vectors`? Or did you want to do something like have a different value than default for one of the pretrained vector aliases (e.g. if you wanted a different unk_init for glove)? 

I agree that it'd be nice to have for the latter case, but a temporary workaround would be to just manually instantiate the Vector class in question and pass these kwargs in (see [this dict](https://github.com/pytorch/text/blob/master/torchtext/vocab.py#L378-L391) for what each alias refers to).

**Comment by jwohlwend** [comment]
Yes, I had the latter in mind. The reason I ran into this was that I was using the pretrained embeddings in two different projects and realized that I was downloading the word vectors twice instead of using a single cache folder. 

I was loading the string alias from a config file, which is why I thought it'd be nice to have that option in the load_vectors function, but you're right there's no particular reason that I can't make my own aliases to instantiate the Vector class with the appropriate kwargs. Thanks for your response!

**Comment by windweller** [comment]
Encountering a similar problem but with `unk_init` argument. For now it seems like the unknown keyword defaults to 0, which is fine, but in certain settings it might be better to randomly initialize unknown keywords.

Before a fix is out, is there an easy way to map unknown keywords to random vector (I can just sweep through....)
